File: Easy/ConsecutiveOnes/ConsecutiveOnes.py
class Solution:
    def solve(self, nums):
        check = 0
        for i in range(len(nums)):
            if nums[i] == 1 and check == 0:
                check += 1
                continue

            if nums[i-1] != 1 and nums[i] == 1 and check > 0:
                return False

        return True

File: Easy/ConsecutiveOnes/ConsecutiveOnes.js
class Solution {
    solve(nums) {
        let check = 0

        for (let i = 0; i < nums.length; i++) {
            if (nums[i] == 1 && check == 0) {
                check++
                continue
            }

            if (nums[i - 1] != 1 && nums[i] == 1 && check > 0 && i > 0) {
                return false
            }
        }

        return true
    }
}